server.go

```diff
@@ -313,11 +318,10 @@ func (s *Server) Handle(ctx context.Context, conn net.Conn) {
 				writeErr = writer.WriteError("Command not supported")
 			}
 		}
-		if command.IsLast() {
+		if parseErr != nil || command.IsLast() {
 			writer.Flush()
 		}
-		if writeErr != nil {
-			// TODO(agis) log these errors
+		if parseErr != nil || writeErr != nil {
```

Comment:
Should we at least log `writeErr`s ?  I think the previous TODO was valid :)

Reply:
They are returned to the clients, so no need to also log them in the server.

Reply:
Added logging.